tools/spy-detector/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "spy-detector"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+nng = "1.0.1"
+dashmap = "6.1.0"
+chrono = "0.4.38"
```

Comment:
I don't think we need to log the timestamps (as noted below) - I think we can drop chrono.

Reply:
correct!!